enhancements/installer/agent-late-binding.md

```diff
@@ -0,0 +1,179 @@
+---
+title: agent-late-binding
+authors:
+  - "@avishayt"
+reviewers:
+  - "@mhrivnak"
+  - "@filanov"
+  - "@ronniel1"
+approvers:
+  - TBD
+creation-date: 2021-05-19
+last-updated: 2021-05-19
+status: implementable
+see-also:
+  - "enhancements/installer/connected-assisted-installer.md"
+---
+
+# Agent Late Binding
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+In the current implementation of Assisted Installer, each Discovery ISO is
+associated with an existing cluster resource (a cluster definition that may not
+yet be an installed OCP cluster).  When an agent comes up, it registers with
+the Assisted Service and is automatically associated with that same cluster
```

Comment:
Here would be a good place to say that the Agent is already "bound" to a cluster.